### PR TITLE
Update vite.config.js fixes #2

### LINKED ISSUE #2: Svelte app returns 502
I am having an issue in a project where one of the proxied node apps is returning a 502, so I tried to reproduce it in your repository and it did exactly the same.

The steps I followed are:

- Download repo
- `ddev start`

The KeystoneCMS works correctly but the one that should be the svelte app returns a 502 from nginx.

Logs on `web`:

```
2024/06/19 03:41:55 [error] 2127#2127: *6 no live upstreams while connecting to upstream, client: 172.19.0.5, server: app.my-project.ddev.site, request: "GET / HTTP/1.1", upstream: "http://localhost/", host: "app.my-project.ddev.site"
```

Not sure if reproducible by anyone, I had a colleague with the same issue on Linux. I have tried this using ddev 1.23.0 - 1.23.2 with Orbstack and Docker Desktop.

### PR DESCRIPTION
**Pull request by elabx**
Update vite server config due to localhost not resolving correctly.

Fixes #2

### CODE REVIEW

File: svelte/vite.config.js [hunk 1]
```diff
@@ -3,4 +3,7 @@ import { defineConfig } from 'vite';
 
 export default defineConfig({
 	plugins: [sveltekit()]
+	server:{
+		host: '127.0.0.1'
```
Comment (rfay) [inline svelte/vite.config.js:7]: I assume that here the host was defaulting to `localhost` ?

Does this fix the problem you had?

**Review by rfay — CHANGES_REQUESTED** [review]
This PR needs a bit more discussion of what the issue was you encountered.

The title should have a description of what it does, and it should have `fixes #2` (it does, right?)

So 
* change the title to explain what's going on, adding `fixes #2` at the end of the title
* In the description (you can edit it) explain what you discovered
* Explain how this fixes it.

File: svelte/vite.config.js (hunk 1 above)
Comment (elabx) [reply svelte/vite.config.js:7]: Yes to both questions.

**Comment by elabx** [comment]
Sorry, I completely lack understanding of how changing to 127.0.0.1 instead of localhost fixed the issue I was having, so will close this for now. I also don't understand if this might just be circumstantial to my particular setup.

**Pull request closed without merging**